src/presenters/teaser-presenter.js

```diff
@@ -366,7 +366,7 @@ const TeaserPresenter = class TeaserPresenter {
 	// returns publishedDate, status, classModifier
 	liveBlog () {
 		return {
-			publishedDate: this.data.updates && this.data.updates[0].date,
+			publishedDate: this.data.updates && Array.isArray(this.data.updates) && this.data.updates[0].date,
```

Comment:
Worth adding `&& this.data.updates[0]`? Considering that's what is undefined. It could be an empty Array?

Reply:
Yeah true, I *think* an empty array would just return undefined by that stage, but that is probably clearer...

Reply:
```